Index.html

```diff
@@ -0,0 +1,309 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>TravelPro</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header>
+      <nav>
+        <div class="logo">
+          <img
+            src="images\logo.png"
+            alt="Pilania Travel Company Logo"
+            width="50"
+          />
+        </div>
+        <div class="nav-items">
+          <ul>
+            <li>Destinations</li>
+            <li>Hotels</li>
+            <li>Flights</li>
+            <li>Bookings</li>
+            <li>Login</li>
+            <button class="btn signup-btn">Sign up</button>
+            <div class="language-dropdown">
+              <select name="lang" id="lang">
+                <option value="English">EN</option>
+                <option value="Hindi">HN</option>
+                <option value="French">FN</option>
+              </select>
+            </div>
+          </ul>
+        </div>
+      </nav>
+    </header>
+    <section class="main">
+      <div class="landing-left-section">
+        <h4>BEST DESTINATIONS AROUND THE WORLD</h4>
+        <h1>Travel, enjoy and live a new and full life</h1>
+        <p>
+          Built Wicket Longer admire do barton vanity itself do in it. <br />
+          Preferred to sportsmen it engrossed listening. Park gate <br />
+          sell they west hard for the.
+        </p>
+        <button class="btn cta-btn">Find out more</button>
+      </div>
+      <div class="landing-right-section">
+        <img src="images\traveller.png" alt="traveller" />
+      </div>
+    </section>
+
+    <section class="category">
+      <div class="category-container">
+        <div class="category-header">CATEGORY</div>
+        <section class="category-content">We Offer Best Services</section>
+      </div>
+      <div>
+        <img src="images\plus.png" alt="plus" />
+      </div>
+    </section>
+
+    <section class="services-list">

```

Comment:
move this section to `<section class="category">`
and update the class names also